src/main/java/org/jabref/gui/copyfiles/CopyFilesTask.java

```diff
@@ -103,6 +103,10 @@ public CopyFilesTask(BibDatabaseContext databaseContext, List<BibEntry> entries,
                 }
             }
             updateMessage(Localization.lang("Finished copying"));
+
+            if (newPath == null) {
```

Comment:
`newPath` is an optional, so it should never be null (for the same reason as for collections). Please init `newPath` as `empty` (and then this if test should not be necessary).

Reply:
Ah thanks! didn't catch that it was an optional